packages/slice-machine/src/features/navigation/NavigationItem.tsx

```diff
@@ -0,0 +1,63 @@
+import { ActionListItem, Tooltip, useMediaQuery } from "@prismicio/editor-ui";
+import Link from "next/link";
+import type { FC, MouseEventHandler, ReactNode, SVGProps } from "react";
+
+type NavigationItemPropsBase = {
+  title: string;
+  active?: boolean;
+  Icon: FC<SVGProps<SVGSVGElement>>;
+  RightElement?: ReactNode;
+};
+
+type NavigationLinkItemProps = NavigationItemPropsBase & {
+  href: string;
+  onClick?: never;
+};
+
+type NavigationButtonItemProps = NavigationItemPropsBase & {
+  href?: never;
+  onClick: MouseEventHandler<Element>;
+};
+
+type NavigationItemProps = NavigationLinkItemProps | NavigationButtonItemProps;
+
+export function NavigationItem(props: NavigationItemProps) {
+  const { title, href, active, Icon, RightElement, onClick, ...otherProps } =
+    props;
+
+  const isCollapsed = useMediaQuery({ max: "medium" });
+
+  const Content = (
+    <ActionListItem
+      textVariant="normal"
+      backgroundColor="transparent"
+      renderStartIcon={() => <Icon width={isCollapsed ? 28 : 32} />}
+      renderEndIcon={() => RightElement}
+      selected={active}
+    >
+      {isCollapsed ? null : title}
+    </ActionListItem>
+  );
+
+  return (
+    <Tooltip content={title} side="right" visible={isCollapsed}>
+      {href !== undefined ? (
```

Comment:
```suggestion
      {href ? (
```

Reply:
it's because of that
<img width="905" alt="Zrzut ekranu 2024-10-25 o 15 59 48" src="https://github.com/user-attachments/assets/98607fe3-f65d-4fcd-bf8b-4cbacabdb566">